data/saas/config/sendgrid_config.yml

```diff
@@ -0,0 +1,94 @@
+saas_config:
+  fides_key: sendgrid_connector_example
+  name: Sendgrid SaaS Config
+  description: A sample schema representing the Sendgrid connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: host
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host:
+      connector_param: host
+    authentication:
+      strategy: bearer_authentication
+      configuration:
+        token:
+          connector_param: api_key
+
+  test_request:
+    method: GET
+    path: /v3/marketing/contacts
+    
+  endpoints:
+  - name: contacts
+    requests:
+      read:
+        method: POST
+        path: /v3/marketing/contacts/search
+        body: "{\"query\": \"email = '<email>'\"}" # do just emails for now. we don't have support for more dynamic query building.
+        param_values:
+          - name: email
+            identity: email
+        data_path: result
+      update:
+        path: /v3/marketing/contacts
+        method: PUT
+        body: '{
+          "contacts" : [
+            {
+              "email": "<email>" ,
+              <masked_object_fields> 
+            }
+          ]  
+        }'
+        param_values:
+          - name: email
+            references:
+              - dataset: sendgrid_connector_example
+                field: contacts.email
+                direction: from
+      delete:
```

Comment:
The guidance we've been following for the SaaS configs is to prefer the `update` endpoint and only define the `delete` endpoint as a fall back. We can remove the delete for this endpoint.